doc/FunctionalTests.md

```diff
@@ -0,0 +1,46 @@
+# Dealing with Functional Tests
+
+This guide aims to provide a basic idea of the overall structure of Functional Tests (FTs) in PowerPointLabs.
+
+## What is a FT?
+FT is an automatic integration test, in which PowerPointLabs add-in is running in PowerPoint application, and without any mock or stub components (unlike in Unit Testing).
+ 
+FT is important to prevent software regression in PowerPointLabs. From time to time, developers keep refactoring the legacy codes. Without adequate test cases, we won’t feel confident when PowerPointLabs is delivered to users. FT is built to help with automating the test processes that is required to ensure the quality of the end-product.
```

Comment:
Possible rephrasing:
<s>From time to time, developers keep refactoring the legacy codes.</s>
Refactoring code happens on a frequent basis.